pkg/updater/resultstore/resultstore.go

```diff
@@ -70,6 +73,117 @@ type processedResult struct {
 	TargetResults   map[string][]*singleActionResult
 }
 
+type timedError struct {
+	millis int64
+	err    error
+}
+type invocationGroup struct {
+	group       string
+	invocations []processedResult
+	errs        map[string]timedError
+}
+
+const (
+	// PartOf is the configuration value key which links to the group invocation ID.
+	PartOf = "EXECUTED_AS_PART_OF_INVOCATION"
+	// Originally indicates the associated non-overview indication.
+	Originally = "Original_test_invocation"
+)
+
+// buildIdentifier may assume its arguments are non-nil.
+type buildIdentifier func(configpb.TestGroup, processedResult) string
+
+var (
```

Comment:
var () is used with many variables at the same time. also `zero` is only used in a single place